### DISCUSSION TITLE
Can i add/delete configs by python code?

### DISCUSSION THREAD

**Discussion by Danilrrrr** [wg-easy/wg-easy#575] [General]
Please help!
i want to write code which can add new config files and send it by telegram bot

can i do it by wg easy or i should try to use another service?

**Comment by Pleonazm** [comment]
It depends on what tools you have, or can use.
The one way with wg-easy (even if crude) could be using something like Selenium to emulate a real user browsing WebUI.

**Comment by eandersons** [comment]
You could utilize API. See #416 for some ideas.
